### DISCUSSION TITLE
How to use it with Angular

### DISCUSSION THREAD

**Discussion by egalley** [antvis/G2#6108] [Q&A]
Hello everybody,

Previously I was using release 4.2.10 of G2.
Now I want to upgrade to 5.1.15.

But I'm facing an error 
Error: node_modules/@antv/component/lib/animation/fadeIn.d.ts:3:85 - error TS2307: Cannot find module '_@antv_g-lite@1.2.21@@antv/g-lite' or its corresponding type declarations.

Even after installation of @antv/g-lite.

Any help would be welcome.

Thanks

Eric

**Comment by hustcc** [comment]
You can refer to [these libraries](https://github.com/antvis/G2/discussions/5772) to develop an Angular wrapper for G2. Once it's developed, we will also include it here.

**Reply by hustcc** [reply]
This error, I suspect, is most likely because there are multiple different versions of `@antv/g` in your project.
